### ISSUE TITLE
container.Base: onInsert => parent view handler

### ISSUE THREAD

**Issue by tobiu** [neomjs/neo#230]
related to #229

parse items of components inside containers for string based listeners / handlers inside the parent view chain.

should happen before the controllers check them and not throw errors in case there are no matches (since those could be inside the controller(s) scope.

**Comment by tobiu** [comment]
already done.
